src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/config.jelly

```diff
@@ -29,6 +29,9 @@
         </f:description>
       </j:if>
     </f:entry>
+    <f:entry title="${%Test branch}" field="mablBranch">
```

Comment:
I wonder if this should be mabl branch. As opposed to source branch. We also branch flows and other things. I am not 100% sure what the best term here would be.

Reply:
I'll change this to mabl branch, thank you.